.github/workflows/publish-new-release.yml

```diff
@@ -65,21 +65,14 @@ jobs:
           echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
       -
         name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3.3.0
```

Comment:
Is there a specific reason why you pin each GH action to a specific version?

Reply:
to be honest this was a teamwork, we have dependabot active and it updates the dependencies for us, and, as long as the CI works we are good with it